File: WizQuixTwitchPaintExtension/Classes/Models/Commands/ClearPaletteCommand.ts
class ClearPaletteCommand extends ACommand
{
    public constructor(client:WebClient)
    {
        super("clearpalette", client);
    }

    public OnInfo(data: string[])
    {
        this.Client.Canvas.Palette.ClearPalette();
    }

    public OnError(code: number, error: string)
    {
        
    }
}

File: WizQuixTwitchPaintApiService/Logic/Commands/ClearPaletteCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WizQuixTwitchPaintApiService.Logic.Commands
{
    /*
        command: clearpalette // this command will clear the palette
        on success: info clearpalette OK
     */
    public class ClearPaletteCommand : ACommand
    {
        public ClearPaletteCommand(WebClient client) : base("clearpalette", client) { }

        public override async Task Execute(string[] args)
        {
            await Client.JoinedRoom.ClearColors();
            await SendInfo();
        }
    }
}
